examples/README.md

```diff
@@ -17,16 +17,16 @@ push to from inside your cluster. If you are following instructions
 
 ```bash
 # To invoke the build-push Task only
-kubectl apply -f examples/taskruns/taskrun.yaml
+kubectl apply -f examples/v1alpha1/taskruns/task-output-image.yaml
```

Comment:
Thanks for this!

Could you modify these to link to `v1beta1`? It's a newer version and I'd much prefer users not be pointed at alpha examples if we can avoid it!